### PR TITLE
docs: add warning banner for beta/experimental components

### PR DESCRIPTION
**Pull request by rfratto**
Change description of experimental components to drop requirement about feature flags.

image

image

### CODE REVIEW

**Review by spartan0x117 — APPROVED** [review]
LGTM!

File: docs/sources/shared/flow/stability/experimental.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+aliases:
+- /docs/agent/shared/flow/stability/experimental/
+headless: true
+---
+
+> **EXPERIMENTAL**: This is an [experimental][] component. Experimental components
+> are subject to breaking changes changes during development, and may be
+> replaced with no equivalent replacement.
```
Comment (erikbaranowski) [inline docs/sources/shared/flow/stability/experimental.md]: do we mean removed with no equivalent?

**Review by erikbaranowski — APPROVED** [review]
LGTM, 1 sanity check wording thing

File: docs/sources/shared/flow/stability/experimental.md (hunk 1 above)
Comment (rfratto) [reply docs/sources/shared/flow/stability/experimental.md]: Oops, yes :)

**Pull request merged**